parts/k8s/kubernetesagentresourcesvmss.t

```diff
@@ -125,6 +125,11 @@
           "osDisk": {
             "createOption": "FromImage",
             "caching": "ReadWrite"
+            {{if UseAgentCustomImage .}}
```

Comment:
Unfortunately this won't help cut down test flakes since tests use the marketplace image.

Reply:
see above - afaics, when provisioning from a marketplace image, premium os disks are used automatically.

Reply:
Interesting - you are right but then on the machine when I try to verify whether this is an SSD, I am not seeing what I expect.
```
[root@ocp-master-38049041-0 ~]# cat /sys/block/sda/queue/rotational 
1
[root@ocp-master-38049041-0 ~]# lsblk
NAME   MAJ:MIN RM  SIZE RO TYPE MOUNTPOINT
fd0      2:0    1    4K  0 disk 
sda      8:0    0   32G  0 disk 
├─sda1   8:1    0   10G  0 part /
└─sda2   8:2    0   22G  0 part /var
sdb      8:16   0   32G  0 disk 
└─sdb1   8:17   0   32G  0 part /mnt/resource
sr0     11:0    1  628K  0 rom  
[root@ocp-master-38049041-0 ~]# uname -r
3.10.0-862.el7.x86_64
```
https://unix.stackexchange.com/a/65602

Is there a different way to figure this out? It's very unfortunate if we already run it :(

Reply:
![ssd](https://user-images.githubusercontent.com/3900799/41056745-6c65534e-69c5-11e8-8030-4291d6ae3c41.png)
:'(